curriculum/challenges/english/08-coding-interview-prep/rosetta-code/knapsack-problem-bounded.md

```diff
@@ -0,0 +1,109 @@
+---
+id: 5a23c84252665b21eecc7ed2
+title: Knapsack problem/Bounded
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+Write a function that solves the knapsack problem that is as follows:
+You are given an array of objects representing items to be put in a knapsack. The objects have 4 attributes: name, pieces (the number of the particular item), weight, and value. The items need to be selected so that the total weight does not exceed the maximum weight and the value is maximized.
```

Comment:
```suggestion
You are given an array of objects representing items to be put in a knapsack. The objects have 4 attributes: name, pieces (the number of the particular item), weight, and value. The items need to be selected so that the total weight does not exceed the maximum weight and the value is maximized.  Keep in mind that each item can appear between 0 and <code>pieces</code> times.
```